pom.xml

```diff
@@ -75,11 +75,14 @@
 		<version.hamcrest>2.2</version.hamcrest>
 		<version.equalsverifier>3.1.11</version.equalsverifier>
 		<version.openpojo>0.8.13</version.openpojo>
+		<version.jacoco>0.8.3</version.jacoco>
 
 		<!-- database driver versions -->
 		<version.h2>1.4.197</version.h2>
 		<version.db2>11.1.1.1</version.db2>
 		<version.postgres>42.2.9</version.postgres>
+		<!-- used by jacoco to collect coverage -->
+		<argLine></argLine>
```

Comment:
Can you explan this a little further or link me a referrence? I don't see why an empty property will help

Reply:
Only initialized with the profile coverage activated, but used by surefire plugin. 
IntelliJ shows this as undefined otherwise.